code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -51,19 +52,26 @@
 	var/treatment_fire = /datum/reagent/medicine/kelotane
 	var/treatment_tox_avoid = /datum/reagent/medicine/tricordrazine
 	var/treatment_tox = /datum/reagent/medicine/charcoal
-	var/treatment_tox_toxlover = /datum/reagent/toxin
+	var/treatment_tox_toxlover = /datum/reagent/toxin //Injects toxins into people that heal via toxins
 	var/treatment_virus_avoid = null
 	var/treatment_virus = /datum/reagent/medicine/spaceacillin
 	var/treat_virus = 1 //If on, the bot will attempt to treat viral infections, curing them if possible.
 	var/shut_up = 0 //self explanatory :)
 
+	var/piercing = FALSE //Do we pierc through clothing?
+	var/hypospray = FALSE //Do we have the nicer needle? - Injects faster
+	var/upgraded_dispenser_1 //Do we have the nicer chemicals? - replaces dex with salbutamol
```

Comment:
please bitflag this or something, infact hell why don't you make a var/medibot_flags variable because all of these booleans are making my head spin (no seriously, there's a performance reason we use bitflags).